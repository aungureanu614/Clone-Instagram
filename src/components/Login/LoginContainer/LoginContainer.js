import React, { Component, Fragment } from 'react';
// import SignInForm from '../SignInForm/SignInForm';
// import RegisterForm from '../RegisterForm/RegisterForm';
import Header from '../../common/Header/Header';
import { imageContainer } from './LoginContainer.module.css';
import image from '../../../icons/ig-logo.png';
import Icon from '../../common/Icon/Icon';

class LoginContainer extends Component {
  constructor() {
    super();
    this.state = {
      isLoginScreen: true,
    };

    this.toggleSignIn = this.toggleSignIn.bind(this);
  }

  toggleSignIn() {
    const { isLoginScreen } = this.state;
    this.setState({
      isLoginScreen: !isLoginScreen,
    });
  }

  render() {
    const { isLoginScreen } = this.state;
    const signInAction = isLoginScreen ? 'Sign In' : 'Create Account';
    return (
      <Fragment>
        <Header
          option={signInAction}
          text="Instagram-Clone"
          icon={<Icon image={image} alt="Instagram icon" className={imageContainer} />}
        />
      </Fragment>
    );
  }
}

export default LoginContainer;
