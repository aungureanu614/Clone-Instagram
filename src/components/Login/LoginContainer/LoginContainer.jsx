import React, { Component, Fragment } from 'react'

import Header from '../../common/Header'
import Icon from '../../common/Icon'

import styles from './LoginContainer.module.css'
import image from '../../../icons/ig-logo.png'

class LoginContainer extends Component {
  state = {
    isLoginScreen: true,
  }

  toggleSignIn = () => {
    const { isLoginScreen } = this.state

    this.setState({
      isLoginScreen: !isLoginScreen,
    })
  }

  render() {
    return (
      <div />
    )
  }
}

export default LoginContainer
