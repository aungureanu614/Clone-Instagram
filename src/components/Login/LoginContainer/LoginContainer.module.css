/* .container {
  box-shadow: 0px 0px 30px 1px #888888;
  display: flex;
  align-items: center;
  width: 768px;
  max-width: 100%;
  min-height: 480px;
} */

/* .overlay-container {
  position: absolute;
  background-image: linear-gradient(red, yellow);
  min-height: 480px;
  width: 768px;
  color: white;
  display: flex;
  align-items: center;
  width: 300px;
}

.overlay {
  display: flex;
  align-items: center;
  justify-content: center;
}

 .overlayPanel {
  display: flex;
  flex-direction: column;
  align-items: center;
  padding: 0 50px;
  text-align: center;
} */