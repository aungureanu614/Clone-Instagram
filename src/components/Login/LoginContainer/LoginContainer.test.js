import React from 'react'
import renderer from 'react-test-renderer'
import LoginContainer from './LoginContainer'

test('LoginContainer renders', () => {
  const component = renderer.create(<LoginContainer />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
