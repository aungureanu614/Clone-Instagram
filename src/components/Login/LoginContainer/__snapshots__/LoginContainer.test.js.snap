// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginContainer renders 1`] = `
<div
  className="headerContainer"
>
  <div
    className="left"
  >
    <img
      alt="Instagram icon"
      className="imageContainer"
      src="ig-logo.png"
    />
    <span>
      Instagram-Clone
    </span>
  </div>
  <div
    className="right"
  >
    Sign In
  </div>
</div>
`;
